src/explore-education-statistics-admin/src/modules/chart-builder/ChartConfiguration.tsx

```diff
@@ -126,6 +130,17 @@ const InfographicChartOptions = ({
       .catch(handleApiErrors);
   }, [releaseId, handleApiErrors]);
 
+  React.useEffect(() => {
```

Comment:
Probably overcomplicating this by using an effect and state. This can either be memoized, or just computed on every render.